types/ErrorResponse.ts

```diff
@@ -0,0 +1,3 @@
+export interface ErrorResponse {
+    error?: string | string[];
+}
```

Comment:
This is something #17 already covers, so you might have a conflict depending on when either gets merged